demisto_sdk/commands/validate/file_validator.py

```diff
@@ -650,6 +651,24 @@ def validate_all_files(self):
         for file in all_files_to_validate:
             self.run_all_validations_on_file(file, file_type=find_type(file))
 
+    def validate_pack(self):
+        """Validate files in a specified pack"""
+        packs = glob(f'{PACKS_DIR}/*')
+        if self.pack_path not in packs:
+            raise Exception(F'File {self.pack_path} was not found\nMake sure you are using relative path')
+        pack_files = {file for file in glob(fr'{self.pack_path}/**', recursive=True) if not os.path.isdir(file)}
+        pack_unique_files_validator = PackUniqueFilesValidator(os.path.abspath(self.pack_path))
```

Comment:
just call `validate_pack_unique_files()` from this class instead, it does exactly that

Reply:
Yeah, but I want to give it a specific path to the pack. 
Else, it will go over all packs. That's why I did it this way

Reply:
it will iterate all the packs you give it, so if you give one pack, which is the one you already got, it will run only on it
i would like to avoid this code duplication as it might cause us issues in the future maintaining it